implementations/rust/ockam/ockam_api/src/cli_state.rs

```diff
@@ -689,6 +689,12 @@ impl NodesState {
 
     pub fn create(&self, name: &str, mut config: NodeConfig) -> Result<NodeState> {
         config.name = name.to_string();
+
+        // check if node name already exists
+        if self.dir.join(name).exists() {
+            return Err(CliStateError::AlreadyExists(format!("node with name `{}` already exists", name)));
```

Comment:
To keep the error consistent, let's change the text to:

```return Err(CliStateError::AlreadyExists(format!("node `{name}`")));```

Reply:
Thank you for the feedback. I made the changes.